src/util/random.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+namespace otto::util {
+
+  struct fastrand_in_range {
+
+    fastrand_in_range(unsigned int seed) : g_seed(seed) {}
+
+    inline int operator()(unsigned int n) { 
+      g_seed = (214013*g_seed+2531011); 
+      return ((g_seed>>16)&0x7FFF) % n; 
+    }
+
+  private:
+    unsigned int g_seed;
```

Comment:
Looks like you didnt end up using this? would be nicer if you did i think...

Reply:
True. I thought it would be nicer to have just a single float in ArpeggiatorState, instead of a full RNG object. Perhaps you disagree?

Reply:
Its the same size anyway

Reply:
I can change it.